docs/guides/admin/docs/configuration/inbox.md

```diff
@@ -0,0 +1,121 @@
+InboxScannerService
+=====================
+
+Overview
+--------
+
+Besides ingesting media packages using the REST service of the IngestService,
+dedicated inbox directories located in the file system can be scanned by
+Opencast. This, for example allows adding media packages to Opencast by copying
+it to a specific location using scripting/SFTP without the need for any HTTP
+traffic. Opencast periodically scans the specified location for new files.
+
+Each directory may result in digest for a different organization or with a
+different default workflow.
+
+Step 1: Configure an InboxScannerService
+-----------------------------------------------
+
+Adjust
+[etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg](`https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg`).
```

Comment:
You probably do not want to use ticks around the link.

Reply:
```suggestion
[etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg](https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg).
```